lti_emailer/canvas_api_client.py

```diff
@@ -43,14 +47,30 @@ def get_courses_for_account_in_term(account_id, enrollment_term_id,
     return result
 
 
-def get_enrollments(canvas_course_id, section_id):
+def get_enrollments(canvas_course_id, section_id=None):
+    """
+    Get the enrollments for a section if a section id is provided.
+    Otherwise get all enrollents for the course.
+    :param canvas_course_id:
+    :param section_id:
+    :return enrollments list:
+    """
     users = get_users_in_course(canvas_course_id)
     enrollments = []
     for user in users:
         for enrollment in user['enrollments']:
-            if enrollment['course_section_id'] == section_id:
+            if section_id:
+                if str(enrollment['course_section_id']) == section_id:
+                    _copy_user_attributes_to_enrollment(user, enrollment)
+                    enrollments.append(enrollment)
+            else:
+                # if the user has any enrollment in the course
+                # we add them to the list and then stop (break)
+                # on to the next user.
                 _copy_user_attributes_to_enrollment(user, enrollment)
                 enrollments.append(enrollment)
+                break
```

Comment:
This if/else can be changed to an if with `if not section_id or str(enrollment['course_section_id']) == section_id:`